techniques/css/C14.html

```diff
@@ -54,70 +52,66 @@
 
 <div class="default-grid with-gap leftcol">
 <aside class="box nav-hack sidebar standalone-resource__sidebar">
-    <header class="box-h">Page Contents</header>
+    <header class="box-h">このページの内容</header>
     <div class="box-i">
       <nav aria-label="page contents" class="navtoc">
-        <ul><li><a href="#technique">About this Technique</a></li>
-<li><a href="#description">Description</a></li>
-<li><a href="#examples">Examples</a></li>
-<li><a href="#resources">Related Resources</a></li>
-<li><a href="#related">Related Techniques</a></li>
-<li><a href="#tests">Tests</a></li>
+        <ul><li><a href="#technique">このテクニックについて</a></li>
+<li><a href="#description">解説</a></li>
+<li><a href="#examples">事例</a></li>
+<li><a href="#resources">関連リソース</a></li>
+<li><a href="#related">関連テクニック</a></li>
+<li><a href="#tests">検証</a></li>
 </ul>
       </nav>
     </div>
 </aside>
 
 <main id="main" class="standalone-resource__main">
-<!-- 日本語訳されるまでの一時的な注記 -->
-<div class="untranslated-note"><p>このWCAG 2.2 テクニック集の日本語訳は作業中となっています。WCAG 2.1 達成方法集の日本語訳をご利用いただけます。<br>[ <a href="https://waic.jp/translations/WCAG21/Techniques/">WCAG 2.1 達成方法集</a> ]</p></div>
 		
-<h1><span>Technique C14:</span>Using <code class="language-css">em</code> units for font sizes</h1>
+<h1><span>テクニック C14:</span>フォントサイズに <code class="language-css">em</code> 単位を使用する</h1>
 
 
 
 
 <section id="technique" class="box">
-	<h2 class="box-h box-h-icon">About this Technique</h2>
+	<h2 class="box-h box-h-icon">このテクニックについて</h2>
 	<div class="box-i">
 
 
   
-  <p>This technique relates to:</p>
+  <p>このテクニックは、次に関連する:</p>
   <ul>
   
     <li>
       
-<a href="https://w3c.github.io/wcag/understanding/resize-text">1.4.4: Resize Text</a>
-(<strong>Sufficient</strong>
-  when used for techniques for relative measurements)
+<a href="https://waic.jp/translations/WCAG22/Understanding/resize-text">1.4.4: テキストのサイズ変更</a>
+(相対長さのテクニックに使用される場合に<strong>十分なテクニック</strong>)
```

Comment:
「相対長さ」という表現が、少し引っかかりました。measurement の訳が難しいのですが、「相対的なサイズ設定のテクニックとして使用される場合に sufficient である」ということかなと思います。
意訳しすぎかもしれませんが...
```suggestion
(相対的なサイズ設定のテクニックとして使用される場合に<strong>十分なテクニック</strong>)
```